packages/subapp-web/lib/init.js

```diff
@@ -62,8 +62,11 @@ ${inlineRuntimeJS}
 </script>`;
 
   // check if any subapp has server side code with initialize method and load them
-  const subAppServers = Object.keys(subappUtil.getAllSubAppManifest())
```

Comment:
Runthis only for subapps for that specific route. In `subapp-server`, we set the subapps for each route to `setupContext.routeOptions.__internals`